docs/source/upgrade.rst

```diff
@@ -8,6 +8,26 @@ This is a list of changes introduced in each release that users should be aware
 Most changes are carefully designed not to break existing code; however changes that may possibly break them are highlighted with a box.
 
 
+Chainer v5
+==========
+
+Changed Behavior of ``Optimizer.new_epoch``
+-------------------------------------------
+
+This change only affects to a minor number of users (who call ``new_epoch()`` method of Optimizers while using a trainer, or who implement their own Updater class).
+
+Optimizers provide ``new_epoch()`` method, which can be used to change optimizer behavior depending on the current epoch number.
+Prior to Chainer v5, this method was expected to be called by users.
+In Chainer v5, Updaters call ``new_epoch()`` method of Optimizers automatically.
+If you are calling ``new_epoch()`` method manually while using a trainer (Updater), you may need any of the following fixes:
+
+* Pass ``auto_new_epoch=False`` to the constructor of the Updater to stop ``new_epoch()`` being called automatically by Updaters.
```

Comment:
* "Pass ... to `Updater.__init__` to stop..." (with a link to `Updater` class)
* ... from being called ...
* Updaters -> the updater

Reply:
> "Pass ... to Updater.__init__ to stop..." (with a link to Updater class)

As `__init__` is not a part of Updater interface, I added link to `StandardUpdater` as an example.

Reply:
"constructor" is not a correct term in Python.